estimagic/logging/read_log.py

```diff
@@ -81,3 +82,56 @@ def read_optimization_iteration(path, iteration, include_internals=False):
             del data[key]
 
     return data
+
+
+def load_start_params(path_or_database):
```

Comment:
rename to `read_start_params` to be more consistent.

Reply:
Done in a675a68 .